packages/dev/core/src/Materials/Textures/multiRenderTarget.ts

```diff
@@ -491,14 +504,15 @@ export class MultiRenderTarget extends RenderTargetTexture {
         const types: number[] = [];
         const samplingModes: number[] = [];
         const useSRGBBuffers: boolean[] = [];
+        const formats: number[] = [];
         const targetTypes: number[] = [];
         const faceIndex: number[] = [];
         const layerIndex: number[] = [];
         const layerCounts: number[] = [];
 
         this._textureNames = textureNames;
 
-        this._initTypes(count, types, samplingModes, useSRGBBuffers, targetTypes, faceIndex, layerIndex, layerCounts, options);
+        this._initTypes(count, types, samplingModes, useSRGBBuffers, formats, targetTypes, faceIndex, layerIndex, layerCounts, options);
         this._multiRenderTargetOptions.types = types;
         this._multiRenderTargetOptions.samplingModes = samplingModes;
         this._multiRenderTargetOptions.useSRGBBuffers = useSRGBBuffers;

```

Comment:
`this._multiRenderTargetOptions.formats = formats;` is missing.

Reply:
my bad